src/components/GoodList.tsx

```diff
@@ -0,0 +1,11 @@
+export const GoodList: React.FC<{ goods: string[] }> = ({ goods }) => (
```

Comment:
Why do you use functional component here and the class component in App?
Use only one thing